mods/furniture/mod.conf

```diff
@@ -0,0 +1,6 @@
+name = furniture
+
+title = Furniture
+description = Provides many furniture nodes
+
+depends = default
```

Comment:
This is only for testing, as we don't have base nodes defined yet. I feel that other mods (which define nodes) should depend on furniture, rather than the other way around.